src/components/common/Header.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from 'react-router-dom';
+import './index.scss';
+
+export function Header () {
+  return (
+    <div className='header'>
+      <div className="header-container">
+        <div className="header--item" style={{marginInlineEnd: 'auto'}}>
+          <Link to="/">
+            <div className="header--item-icon" style={{backgroundImage: 'url(../images/home.png)'}}></div>
+          </Link>
+        </div>
+        <div className="header--item">
+          <Link to="/catalog">
+            <div className="header--item-icon" style={{backgroundImage: 'url(../images/catalog.png)'}}></div>
```

Comment:
check inline styles please and move them to class when you don't have additional logic